src/add/operator/buffer.ts

```diff
@@ -4,4 +4,8 @@ import {buffer} from '../../operator/buffer';
 
 Observable.prototype.buffer = buffer;
 
-export var _void: void;
\ No newline at end of file
+declare module '../../Observable' {
+  interface Observable<T> {
+    buffer: (closingNotifier: Observable<any>) => Observable<T[]>;
```

Comment:
It'd be great if it works, but in that case let's go with as-is and try to find better way in further.